src/main.js

```diff
@@ -81,13 +87,23 @@ export default class LevelSelector extends UICorePlugin {
       this.$el.html(this.template({ 'levels':this.levels, 'title': this.getTitle() }))
       this.$el.append(style)
       this.core.mediaControl.$('.media-control-right-panel').append(this.el)
+      this.$('.level_selector ul').css('max-height', this.core.el.offsetHeight*0.8)
       this.highlightCurrentLevel()
     }
     return this
   }
 
   fillLevels(levels, initialLevel = AUTO) {
     if (this.selectedLevelId === undefined) this.selectedLevelId = initialLevel
+
+    let onLevelsAvailable = this.core.options.levelSelectorConfig.onLevelsAvailable
```

Comment:
Maybe, we need to add more resilience here? Like:
```suggestion
    let onLevelsAvailable = this.core.options && this.core.options.levelSelectorConfig && this.core.options.levelSelectorConfig.onLevelsAvailable
```